sympy/series/order.py

```diff
@@ -415,6 +416,10 @@ def _eval_subs(self, old, new):
                         from sympy.solvers.solveset import solveset
                         d = Dummy()
                         sol = solveset(old - new.subs(var, d), d)
+                        if isinstance(sol, Complement):
```

Comment:
Yes, this is already tested by `two tests`. One of them being `sin(1/x).series(x, oo, n=5)`, in the `sympy.core.tests.test_wester.py` .